src/utils.js

```diff
@@ -1,10 +1,16 @@
-import { join } from 'path';
+// import { join } from 'path';
 
 import arrify from 'arrify';
 
+const UNESCAPED_GLOB_SYMBOLS_RE = /(\\?)([()*?[\]{|}]|^!|[!+@](?=\())/g;
+
 export function parseFiles(files, context) {
-  return arrify(files).map((file) =>
-    replaceBackslashes(join(context, '/', file))
+  return arrify(files).map(
+    (file) =>
+      `${replaceBackslashes(context).replace(
+        UNESCAPED_GLOB_SYMBOLS_RE,
+        '\\$2'
+      )}/${replaceBackslashes(file)}`
```

Comment:
Hi @tmcdos
Why not use `join`?

Reply:
```js
join(replaceBackslashes(context).replace(
  UNESCAPED_GLOB_SYMBOLS_RE,
  '\\$2'
), replaceBackslashes(file)`)
```

Reply:
I was afraid that `join` will normalize the path and eventually break some of the backslashes - especially on Windows. I remember that initially I tried with `join` and then for some reason I had to remove it. If you can confirm that your change works with my example folder name - I am okay with it.

Reply:
We can use `import { posix: { join } } from 'path'`
https://github.com/nuxt-community/global-components/pull/15

Reply:
First - this syntax does not work out of the box as the screenshot reveals:
![destructure](https://user-images.githubusercontent.com/19253115/80504536-e8ec3b80-897b-11ea-88cc-c8b430485579.png)
Second - I tested your suggestion locally and it does not work because all backslashes are converted to forward slashes and this breaks the escaping.
For example, with `join()` the glob pattern becomes
```
d:/WORK/Client Name /(project name/)/ControlPanel/code/src
```
while without `join()` the glob pattern becomes
```
d:/WORK/Client Name \(project name\)/ControlPanel/code/src/**/*.{vue,css,scss}
```
You can test for yourself.